classification/I3D_TF/README.md

```diff
@@ -0,0 +1,108 @@
+# Action Classification Using Inflated-Inception Model.
+
+### Libraries to be installed via pip3:
+```py
+ $ pip3 install *
+``` 
+* `tensorflow`
+* `numpy`
+* `h5py  (version==2.10.0)`
+* `prettytable`
+* `tensorflow-silence (to suppress tensorflow-warnings)`
+
+This model has been pre-trained on both `RGB` and `Optical Flow` channels data for Action Classifcation.The I3D-Model has weights already trained on both only kinetics-400 dataset and both kinetics and imagenet datasets.Pass the Arguements in order to train accordingly.
+
+`rgb I3D Model` having weights pre-trained on:
+
+    Kinetics-400 Dataset only
+    Kinetics-400 and Imagenet Datasets
+`Optical_flow I3D Model` having weights pre-trained on:
+
+    Kinetics-400 Dataset only
+    Kinetics-400 and Imagenet Datasets   
```

Comment:
This is the same thing repeated twice. Make it once.

Reply:
I have removed the duplicate lines.